src/screens/ProfileScreen/elements/FollowInfoRow.js

```diff
@@ -18,15 +18,21 @@ import {getSingularOrPluralText} from '../../../utils/string/StringUtils';
  * @property {String} follower
  * @property {String} following
  * @property {OnFollowingContainerClicked} onFollowingContainerClicked
+ * @property {onFollowersContainerClicked} onFollowersContainerClicked
  */
 /**
  *
  * @param {FollowInfoRowPropsParam} param0
  * @returns
  */
-const FollowInfoRow = ({follower, following, onFollowingContainerClicked}) => {
+const FollowInfoRow = ({
+  follower,
+  following,
+  onFollowingContainerClicked,
+  onFollowersContainerClicked
+}) => {
   const openFollower = () => {
-    SimpleToast.show('For privacy reasons, you cannot see who follows you', SimpleToast.LONG);
+    onFollowersContainerClicked();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `onFollowersContainerClicked` is not properly documented. It's missing the type definition in the JSDoc comment. Please add a type definition for this prop.
```diff
  * @property {OnFollowingContainerClicked} onFollowingContainerClicked
+ * @property {Function} onFollowersContainerClicked
   */
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->